src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.test.tsx

```diff
@@ -17,4 +17,45 @@ describe('<ProductTemplate />', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it('renders without title', () => {
+    const { container } = render(
+      <ProductTemplate prevStep="prevStep" progress={{ currentStep: 2, totalSteps: 4 }} contentWidth={10}>
+        <ProductTemplate.Card>This is the body of the card</ProductTemplate.Card>
+      </ProductTemplate>,
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
We are moving away from creating snapshots for everything as it slows down the tests. 

If you want to check if it renders without title the correct approach is to get the title and check is not there. 

If you want to check specific style changes you can also you use the assets `jest-styled-components` is providing like [toHaveStyleRule](https://github.com/styled-components/jest-styled-components#tohavestylerule)

Reply:
Got it. Was just following the apparent convention